packages/windows/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["src/**/*"],
+  "compilerOptions": {
+    "outDir": "./lib",
+    "rootDir": "./src",
+    "paths": {
+      "@navaiguide/core": ["./packages/core/lib"]
```

Comment:
Curious is this needed, given this whole repo is single workspace...

Reply:
The core project?